### ISSUE TITLE
Error with tournament selection

### ISSUE THREAD

**Issue by aahaselgrove** [wildart/Evolutionary.jl#40]
Trying to use tournament selection gives the error: MethodError: no method matching Array{Int64,N} where N(::Int64)

**Comment by wildart** [comment]
First, create the selector
```julia
s = tournament(2) # parameter is the size of the group of individuals
``
then call the selector with fitness and number of selected individuals
```julia
s([1,2,3], 5) # fitness of population of size 3, select 5 winners from randomly assigned  group of 2 individuals
```
See tests for more examples.

**Comment by aahaselgrove** [comment]
Looks like you fixed it in d4f46f5
